.remake/component/remake-name/remake-name.vue

```diff
@@ -0,0 +1,11 @@
+<template></template>
```

Comment:
```suggestion
<template>
	<div />
</template>
```

Could we populate these components with an empty `div`, so by default they pass lint checking?

Reply:
fine by me